k8s/upgrades/README.md

```diff
@@ -77,29 +77,29 @@ the openebs-operator.yaml for your cluster, you will have to download the
 desired openebs-operator.yaml and customize it again**
 
 ```
-#Upgrade to OpenEBS control plane components to desired version. Say 1.11.0
-$ kubectl apply -f https://openebs.github.io/charts/openebs-operator-1.11.0.yaml
+#Upgrade to OpenEBS control plane components to desired version. Say 1.12.0
+$ kubectl apply -f https://openebs.github.io/charts/openebs-operator-1.12.0.yaml
 ```
 
-### Upgrade using helm chart (using stable/openebs, openebs-charts repo, etc.,):
+### Upgrade using helm chart (using openebs/openebs, openebs-charts repo, etc.,):
 
 **The sample steps below will work if you have installed openebs with
-default values provided by stable/openebs helm chart.**
+default values provided by openebs/openebs helm chart.**
 
 Before upgrading via helm, please review the default values available with
-latest stable/openebs chart.
-(https://raw.githubusercontent.com/helm/charts/master/stable/openebs/values.yaml).
+latest openebs/openebs chart.
+(https://openebs.github.io/charts).
```

Comment:
The link should be: https://github.com/openebs/charts/blob/master/charts/openebs/values.yaml